cfme/scripting/sprout.py

```diff
@@ -41,14 +41,20 @@ def main():
 @click.option('--populate-yaml', is_flag=True, default=False,
               help="Populate the yaml with the appliance")
 @click.option('--provider', default=None, help="Which provider to use")
+@click.option('--provider_type', default=None, help='A provider type to select from')
+@click.option('--template-type', default=None, help='A template type')
```

Comment:
This should look like:

`@click.option('--template-type', 'template_type', default=None, help='A template type')`

Based on line 48.